packages/p2p-v2/src/components/Modals/NicknameModal/__tests__/NicknameModal.spec.tsx

```diff
@@ -36,20 +52,11 @@ jest.mock('@deriv/api-v2', () => ({
     },
 }));
 
-jest.mock('@/hooks', () => ({
-    ...jest.requireActual('@/hooks'),
-    useDevice: jest.fn().mockReturnValue({
-        isDesktop: true,
-        isMobile: false,
-    }),
-    useSwitchTab: jest.fn().mockReturnValue(jest.fn),
-}));
-
 describe('NicknameModal', () => {
     it('should render title and description correctly', () => {
         render(<NicknameModal isModalOpen setIsModalOpen={jest.fn()} />, { wrapper });
-        expect(screen.getByText('Choose your nickname')).toBeVisible();
-        expect(screen.getByText('This nickname will be visible to other Deriv P2P users.')).toBeVisible();
+        expect(screen.getByText('What’s your nickname?')).toBeVisible();
```

Comment:
now that the modal is using the one from deriv-com/ui, do we still need the wrapper?